pkg/constant/constant.go

```diff
@@ -21,5 +21,6 @@ const (
 	ControllerRestoreFinalizer      = "restore-finalizer"
 
 	PluginCSIPVCRestoreRIA            = "velero.io/csi-pvc-restorer"
+	PluginCSIPVCVSDURestoreRIA            = "velero.io/csi-pvc-vs-dv-restorer"
```

Comment:
The alignment is off on the `=` -- also, we need this new plugin to be run *before* "velero.io/csi-pvc-restorer" so when the csi-pvc-restorer is run the VS, DU, etc. are already restored. Maybe change PluginCSIPVCRestoreRIA to "velero.io/csi-pvc-restorer2" and then set PluginCSIPVCVSDURestoreRIA to "velero.io/csi-pvc-restorer1"

Reply:
git it ^ :)  alphabetic order